projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/methods-and-properties.migration.ts

```diff
@@ -1311,6 +1721,27 @@ export const METHODS_AND_PROPERTIES_DEPRECATIONS_DATA: MethodPropertyDeprecation
       comment:
         "// TODO:Spartacus - Property 'numberOfEntriesBeforeAdd' was removed from Class 'AddedToCartDialogComponent'. ",
     },
+    {
+      class: 'AddressBookComponent',
+      importPath: '@spartacus/storefront',
+      deprecatedNode: 'getCardContent',
+      comment:
+        "The 'getCardContent' method's signature changed to: 'getCardContent( address: Address): Observable<{ role: string; textBold: string; text: string[]; actions: { name: string; event: string; }[]; header: string; deleteMsg: string; label: string; }>'",
```

Comment:
another case of implicit typing. Method from 4.3 and 5.0 does not have an explicit return type, but the returned value has additional properties.

I would say this is good that it caught it, but the previous case was different, where it wasn't really a problem.

Reply:
This one is tricky.  Let's discuss if it's still somewhat useful to list those cases.

Reply:
We say it's good to have these cases documented. Not 100% sure it's a breaking change, but good to document regardless.